mapiproxy/servers/default/emsmdb/oxcfxics.c

```diff
@@ -2570,12 +2581,20 @@ _PUBLIC_ enum MAPISTATUS EcDoRpc_RopSyncImportDeletes(TALLOC_CTX *mem_ctx,
 	owner = emsmdbp_get_owner(synccontext_object);
 	openchangedb_get_MailboxReplica(emsmdbp_ctx->oc_ctx, owner, &repl_id, &replica_guid);
```

Comment:
I think this is out of the scope of this PR. I've only added to the one that it was required (no function was used). I'm pretty ok to introduce that change but in another PR. My reasoning is to perform the tests against those changes and having branches in git is cheap and PRs even more. If you are unconfortable with that reasoning, I will modify this PR with those changes but with new commits.